vm-virtio/src/rng.rs

```diff
@@ -325,3 +370,30 @@ impl VirtioDevice for Rng {
         ))
     }
 }
+
+impl Pausable for Rng {
```

Comment:
Every device looks like it has the same implementation. Would it be possible to put it at a higher level?

Reply:
I used a couple of macros to factorize all this, thanks.